artemis-server/src/main/java/org/apache/activemq/artemis/core/persistence/AddressStatus.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 The Apache Software Foundation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.activemq.artemis.core.persistence;
+
+public enum AddressStatus {
```

Comment:
I'd recommend re-using `QueueStatus` and renaming it `AddressQueueStatus` since the two objects are essentially identical to each other.